app/src/main/java/org/greenstand/android/TreeTracker/models/messages/network/MessagesApiService.kt

```diff
@@ -0,0 +1,12 @@
+package org.greenstand.android.TreeTracker.models.messages.network
+
+import org.greenstand.android.TreeTracker.models.messages.network.responses.MessagesResponse
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface MessagesApiService {
+
+    @GET("messaging/message")
+    suspend fun getMessages(@Query("author_handle") wallet: String): MessagesResponse
```

Comment:
Oh great this explains why there was no need for request bodies.

Reply:
What do you mean?